code401challenges/src/test/java/code401challenges/LibraryTest.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package code401challenges;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class LibraryTest {
+    @Test public void testSomeLibraryMethod() {
+        Library classUnderTest = new Library();
+        assertTrue("someLibraryMethod should return 'true'", classUnderTest.someLibraryMethod());
+    }
+    @Test public void test_insertShiftArray(){
```

Comment:
no points off this time, but in the future, I'd like to see more tests, specifically some edge cases. Ex. what would happen if the given was an epmty array, odd vs. even array length, a couple different examples. I want to know that it doesn't just work with this one test, but not others